test/functional/rpc_packages.py

```diff
@@ -356,5 +357,33 @@ def test_submitpackage(self):
         assert_equal(res["tx-results"][sec_wtxid]["error"], "version")
         peer.wait_for_broadcast([first_wtxid])
 
+    def test_maxfeerate_maxburn_submitpackage(self):
+        node = self.nodes[0]
+        # clear mempool
+        deterministic_address = node.get_deterministic_priv_key().address
+        self.generatetoaddress(node, 1, deterministic_address)
+
+        self.log.info("Submitpackage maxfeerate arg testing")
+        chained_txns = self.wallet.create_self_transfer_chain(chain_length=2)
+        chain_hex = [t["hex"] for t in chained_txns]
+        pkg_result = node.submitpackage(chain_hex, 0.00000001)
```

Comment:
Maybe instead of magic number, calculate/specify the parent feerate and make `maxfeerate` a little bit less?

Reply:
done